### PR TITLE
core dump tutorial: add sanity check idea for Fargate EFS setup

### PR DESCRIPTION
**Pull request by PettitWesley**
Signed-off-by: Wesley Pettit <wppttt@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: troubleshooting/tutorials/remote-core-dump/README.md [hunk 1]
```diff
@@ -196,6 +196,8 @@ Then, you can define the volume in the task definition to be your EFS filesystem
     }],
 ```
 
+As a sanity check that your setup the EFS filesystem correctly on the Fargate task, create a file in the EFS filesystem. For example, `touch my-efs-id.txt`. Then, when you later setup your Fargate task, you can use ECS Exec to check that you can see the file in the cores directory: `ls /cores`.
```
Comment (matthewfala) [inline troubleshooting/tutorials/remote-core-dump/README.md:199]: Grammar issue
"As a sanity check that **you** setup"

**Review by matthewfala — CHANGES_REQUESTED** [review]
Minor grammar issue

File: troubleshooting/tutorials/remote-core-dump/README.md (hunk 1 above)
Comment (PettitWesley) [reply troubleshooting/tutorials/remote-core-dump/README.md:199]: fixed

**Review by matthewfala — APPROVED** [review]
Looks good!

**Pull request merged**